src/pages/ContentPage/ContentPage.tsx

```diff
@@ -25,11 +24,8 @@ interface PageHeaderProps extends _PageHeaderProps {
 
 const PageHeader = _PageHeader as FunctionComponent<PageHeaderProps>;
 
-const SamplePage = () => {
+const ContentPage = () => {
   const classes = useStyles();
-  useEffect(() => {
-    insights?.chrome?.appAction?.('view-list-page');
-  }, []);
```

Comment:
**Removing this** and placing it higher in the tree fixes the infinite calls the chrome viewlistpage